integration/proxy/teleterm_test.go

```diff
@@ -197,6 +198,7 @@ func testGatewayCertRenewal(ctx context.Context, t *testing.T, params gatewayCer
 		CreateTshdEventsClientCredsFunc: func() (grpc.DialOption, error) {
 			return grpc.WithTransportCredentials(insecure.NewCredentials()), nil
 		},
+		ClientCache:    clientcache.NewNoCache(storage),
```

Comment:
Hmm, do we want to leave a comment why `no cache` is used?

Reply:
You guys are welcome to do this in a follow up PR, but I had to merge this as it was killing the team's productivity and ability to get a v15 release out.